src/hooks/useScrollable.ts

```diff
@@ -19,6 +19,7 @@ export const useScrollable = () => {
   );
   const isScrollableRefreshable = useSharedValue<boolean>(false);
   const isScrollableLocked = useSharedValue<boolean>(true);
+  const isScrollEnded = useSharedValue<boolean>(true);
```

Comment:
Before this PR, `isScrollEnded` was initialized to `false`, but now it's initialized to `true`.  It seems to make more sense as `true`, but I'm worried that might cause some issues with the existing code unrelated to the `isScrollableLocked` logic.

Reply:
it's only used in scroll handlers and being set on drag begins, so I think the initial value doesn't really matter here https://github.com/discord/react-native-bottom-sheet/blob/discord-fork-4.5.1/src/hooks/useScrollEventsHandlersDefault.ts#L106